ramls/types/resources/resourcePutDataAttributes.json

```diff
@@ -0,0 +1,110 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Resource Put Data attributes schema",
+  "description": "Resource Put Data attributes schema",
+  "type": "object",
+  "javaType": "org.folio.rest.jaxrs.model.ResourcePutDataAttributes",
+  "additionalProperties": false,
+  "properties": {
+    "url": {
+      "type": "string",
+      "description": "URL of a custom resource",
+      "example": "https://hello.io"
+    },
+    "name": {
+      "type": "string",
+      "description": "Name for a custom resource",
+      "example": "Sample Title"
+    },
+    "isPeerReviewed": {
+      "type": "boolean",
+      "description": "Peer review indicator",
+      "example": true
+    },
+    "publicationType": {
+      "type": "string",
+      "description": "Publication Type",
```

Comment:
In description for various fields -- can we add indications about the use of the attribute only for custom resources as is noted in current mod-kb-ebsco raml? `Note that this attribute can be updated ONLY FOR A CUSTOM RESOURCE.` This is a really useful feature in the documentation 
![screen shot 2018-11-13 at 12 14 22 pm](https://user-images.githubusercontent.com/19415226/48430647-cd688300-e73d-11e8-842a-2f6342c9ec90.png)

Reply:
thanks, updated